connector/src/yang/connector/netconf.py

```diff
@@ -414,6 +432,29 @@ def disconnect(self):
 
         self.session.close()
 
+    def notify_wait(self, steps):
```

Comment:
`steps` is a concept in pyATS test cases. Is it possible to avoid passing in `steps`?

Reply:
No, I haven't seen another alternative from my current standpoint.